### DISCUSSION TITLE
i dont have 023000EE for A5 2018

### DISCUSSION THREAD

**Discussion by Loriskjei** [Mr-MIBonk/M.I.B._More-Incredible-Bash#41] [Q&A] [answered]
hello I followed the procedure to the letter, I installed the mib on the green menu but when I check the fec I don't have the correct one 023000eefor unlimited maps but only the one there.

what is the right log for A5 2018?
MHI2Q_ER_AUG22_P5092
mu: 1329

**Comment by Doobys** [comment]
hello, I have the same problem.
Did you find a solution?

**Reply by Doobys** [reply]
Ok, I found the solution...
On MIB in GEM menu, (whith AIO patches in the patches folder), go and clic on "Flash patched image/add FEC/ CP..."

Thank you for this incredible MIB

**Reply by Loriskjei** [reply]
I've tried this step before but it doesn't work

**Comment by harman-f** [comment]
Please share your log files here.

**Comment by Loriskjei** [comment]
where can I find it?

**Reply by harman-f** [reply]
on your M.I.B SD card /backup/logs/

**Comment by Loriskjei** [comment, marked as answer]
I tried to reinstall the AIO but it does not install the FEC 023000ee

**Reply by harman-f** [reply]
change addfec.txt and run addFec again from GEM

**Reply by Loriskjei** [reply]
I did not find any Logs.txt files in the backup folder.
however I realized that I have not downloaded the patch.7z file which must be copied to the patch folder. I hope now the problem is solved

**Reply by harman-f** [reply]
solved?

**Reply by Loriskjei** [reply]
yes it was solved ... I realized that I did not insert the patch in the appropriate folder of the m.i.b ..... it was not clear from the guide that the patch was not included in the mib but you had to download it separately and then insert it in the folder right

**Reply by harman-f** [reply]
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/Prepare-M.I.B-SD-card-&-install-M.I.B#add-patch-files
